src/foam/nanos/column/NestedPropertiesExpression.js

```diff
@@ -37,8 +37,35 @@ foam.CLASS({
     }
   ],
   methods: [
-    function set(o, val) {
-      o.cls_.getAxiomByName(this.nestedProperty).set(o, val);
+    {
+      name: 'set',
+      args: [
+        {
+          name: 'o',
+          type: 'Object'
+        },
+        {
+          name: 'value',
+          type: 'Object'
+        }
+      ],
+      code: function(o, val) {
+        if ( this.nestedProperty.includes('.') ) return; 
```

Comment:
not building FObjectProperty and Reference property obj's here as if one of the properties in chain is reference property we will not be able to set object